lib/src/themes/paint_factory.dart

```diff
@@ -64,11 +66,27 @@ class PaintFactory {
         whenNull: () => LiteralExpression(1.0));
     final strokeWidth = expressionParser.parse(paint['$prefix-width'],
         whenNull: () => LiteralExpression(defaultStrokeWidth));
+
+    List<num> dashArray = [];
+    final dashJson = paint['$prefix-dasharray'];
+    if (dashJson != null) {
+      // check if at least 2 values are specified (otherwise dashing useless)
+      if ((dashJson as List<num>).length >= 2) {
```

Comment:
Rather than `num` could we use `double`? That way we can do all conversions when parsing the map theme instead of every time we render. I've seen number conversions show up in the profiler, surprisingly.

Reply:
Yep makes sense, I'll have a look.

Reply:
done.